.github/workflows/build_wheels.yml

```diff
@@ -1,12 +1,10 @@
 name: Build and upload to PyPI
 
-# Build on every branch push, tag push, and pull request change:
-#on: [pull_request]
-# Alternatively, to publish when a (published) GitHub Release is created, use the following:
+# Publish when a new GitHub version is tagged:
 on:
-  release:
-    types:
-      - published
+  push:
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+'
```

Comment:
I trust you for the regex :)
I use `'v*'` for the other workflows but it should give the same result.

Reply:
Haha I took it form here:
https://stackoverflow.com/a/71076957
Feel free to change to `'v*'` if you prefer.